.github/workflows/osv-scanner.yml

```
@@ -33,6 +33,12 @@ jobs:
         -r
         --skip-git
         ./
+      steps:
+        - name: Upload scan results
```

actionlint (1.7.4)

expected scalar node for string value but found sequence node with "!!seq" tag

(syntax-check)